Fix-Website/app/app.js

```diff
@@ -34,25 +32,44 @@
     app.directive("infoZone", function () {
         return {
             restrict: 'E',
-            templateUrl: "Templates/info.html",
-            controller: function () {
-                this.form = false;
-                this.clickProduct = function () {
-
-                    this.form = !this.form;
-                };
-            },
-            controllerAs: 'info'
+            templateUrl: "Templates/info.txt"
```

Comment:
It is fine for now, but one of the things we'll want to do once the basics of the website are in place are normalize file names.

We should standardize on the files and folders related to this website being all lower-case. e.g. `templates/info.txt`. When you deploy this website to a Linux webserver, random things will randomly stop working because of a case-sensitive file system.